components/_meta/_01-foot.twig

```diff
@@ -2,12 +2,20 @@
   <!--DO NOT REMOVE-->
   {{ patternLabFoot | raw }}
 
-  <!-- Drupal-specific files - UNCOMMENT AS NECESSARY -->
-  <!-- may need to change these below based on path -->
-  <!-- <script src="../../../../../../../core/assets/vendor/domready/ready.min.js"></script>
-  <script src="../../../../../../../core/misc/drupal.js"></script> -->
-  
+  <!-- Drupal-specific usage -->
+  <!-- example using symlinking -->
+  <!-- cd web/themes/custom/emulsify/components/js -->
+  <!-- ln -s ../../../../../core/assets/vendor/domready/ready.min.js ./ -->
+  <!-- ln -s ../../../../../core/misc/drupal.js ./ -->
```

Comment:
Lines 5-9, since they are all the same block and aren't commented out, could these be one comment block?

```
<!-- 
Drupal-specific usage — example using symlinking
`cd web/themes/custom/emulsify/components/js`
`ln -s ../../../../../core/assets/vendor/domready/ready.min.js ./ `
`ln -s ../../../../../core/misc/drupal.js ./`
-->
```